packages/loot-core/src/server/budget/template-notes.ts

```diff
@@ -9,6 +9,7 @@ import {
   resetCategoryGoalDefsWithNoTemplates,
 } from './statements';
 import { Template } from './types/templates';
+import { useTranslation } from 'react-i18next';
```

Comment:
react-i18next doesn't work outside of React

```suggestion
import { t } from 'i18next';
```